package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "devDependencies": {
+    "@nilfoundation/evm-placeholder-verification": "github:NilFoundation/evm-placeholder-verification#17-calldata-gas-marshalling-reduce",
```

Comment:
Is the branch `17-calldata-gas-marshalling-reduce` permanent or will it be someday soon? Using dependencies from dev branches is dangerous. You really have no idea when it'll break, but it definitely will.

Reply:
Aware of this issue , will be handled with 17 merges with master.

Reply:
Fixed by updating the underlying package version.